buildbot-conf/cordova.conf

```diff
@@ -0,0 +1,326 @@
+    NPMInstall(what='cordova-cli',         workdir='cordova-cli'),
+    NPMInstall(what='cordova-js',          workdir='cordova-js'),
+    NPMInstall(what='cordova-plugman',     workdir='cordova-plugman'),
+    NPMInstall(what='platform',            workdir=I('cordova-%(prop:platform)s')),
+    NPMInstall(what='cordova-mobile-spec', workdir='cordova-mobile-spec/createmobilespec'),
+
+    # link the installed code
+    SH(command=['cordova-coho/coho', 'npm-link'], description='coho link'),
+
+    # prepare the test app
+    SH(
+        command = [
+            'node',
+            'cordova-mobile-spec/createmobilespec/createmobilespec.js',
+            I('--%(prop:platform)s')
+        ],
+        description='creating mobilespec app'
+    ),
+    SH(
+        command=[
+            'node',
+            'cordova-medic/updateconfig.js',
+            I('--%(prop:platform)s'),
+        ],
+        description='preparing mobilespec app'
+    ),
+]
+
+cordova_plugins_all = BuildFactory()
+cordova_plugins_all.addSteps(common_setup_steps)
+cordova_plugins_all.addSteps(common_plugins_steps)
+cordova_plugins_all.addSteps([
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js')], description='running tests'),
+])
+
+# WORKAROUND:
+#            this is here to match what medic already does; these
+#            should be their own builders, using a proper test matrix
+cordova_plugins_windows = BuildFactory()
+cordova_plugins_windows.addSteps(common_setup_steps)
+cordova_plugins_windows.addSteps(common_plugins_steps)
+cordova_plugins_windows.addSteps([
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js'), "--store80"], description='running tests (Windows 8.0)'),
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js'), "--store"], description='running tests (Windows 8.1)'),
+    SH(command=['node', I('cordova-medic/build_%(prop:platform)s.js'), "--phone"], description='running tests (Windows Phone 8.1)'),
+])
+
+c['builders'].extend([
+
+    # plugins builders
+    BuilderConfig(name='cordova_plugins_ios_osx',            slavenames=OSX_SLAVES,     factory=cordova_plugins_all,     properties={'platform': 'ios',        'slaveos': OSX}),
+    BuilderConfig(name='cordova_plugins_android_osx',        slavenames=OSX_SLAVES,     factory=cordova_plugins_all,     properties={'platform': 'android',    'slaveos': OSX}),
+    BuilderConfig(name='cordova_plugins_windows_windows',    slavenames=WINDOWS_SLAVES, factory=cordova_plugins_windows, properties={'platform': 'windows',    'slaveos': WINDOWS}),
+    BuilderConfig(name='cordova_plugins_wp8_windows',        slavenames=WINDOWS_SLAVES, factory=cordova_plugins_all,     properties={'platform': 'wp8',        'slaveos': WINDOWS}),
+    BuilderConfig(name='cordova_plugins_android_windows',    slavenames=WINDOWS_SLAVES, factory=cordova_plugins_all,     properties={'platform': 'android',    'slaveos': WINDOWS}),
```

Comment:
The `plugins` is there because we can have `platform` or `tools` instead in future builders. I think imposing structure is very beneficial to disambiguate the builders on the UI. For example, the old naming had `android` and `android-win`, which can be ambiguous if we also supported Linux. In contrast, having `android-win`, `android-osx`, and `android-linux` would be unambiguous.

Also, if iOS Cordova builds ever become possible on Windows, the assertion that `ios_isx` is superfluous will be false.